.github/workflows/dev-testing.yml

```diff
@@ -70,7 +75,7 @@ jobs:
         id: cache-ephys-datasets
         with:
           path: ./ephy_testing_data
-          key: ephys-datasets-2024-03-27-${{ matrix.os }}-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
+          key: ephys-datasets-2024-06-21-ubuntu-latest-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
```

Comment:
What's going on with this change. Had we been storing 3 copies of the same data in the cache?

Reply:
This action only runs in ubuntu-latest:

https://github.com/catalystneuro/neuroconv/blob/cb8687f6456dea51545144bb2e81a0ee4fe36c02/.github/workflows/dev-testing.yml#L20-L22

{{matrix.os}} is not defined here and it was working because the cache action has implicit mechanism to match close strings I think.

I made it explicit to avoid any surprises.